shadow-cljs.edn

```diff
@@ -1,12 +1,33 @@
 ;; shadow-cljs configuration
 {:source-paths
- ["shared/src"]
+ ["shared/src"
+  "shared/test"]
 
+ ;; TODO -- share deps with project.clj -- see https://shadow-cljs.github.io/docs/UsersGuide.html#Leiningen
  :dependencies
- []
+ [[org.clojure/core.match "0.3.0"]
+  [medley "1.3.0"]
+  [net.cgrand/macrovich "0.2.1"]
+  [prismatic/schema "1.1.12"]
+
+  ;; new stuff
+  [lambdaisland/glogi "1.0.83"]]
+
+ :nrepl
+ {:middleware
+  [cider.nrepl/cider-middleware
```

Comment:
to do this i think you'll need to add these dependencies above. i suspect this only works because you have these in your profiles.clj. On other machines I expect this to fail or maybe "gracefully" fail to put these middleware in the handler stack.

Reply:
The deps are in package.json

Reply:
what's the reason for splitting it like that? Also you can omit piggieback from all of this. Shadow implements its own (better) version of piggieback and its unnecessary here.

Reply:
I just didn't want them accidentally ending up in the production build somehow... I know tree-shaking will probably fix all of that anyway, just wanted to be extra safe tho

Reply:
shadow cannot add anything to our jar beyond its compiled artifact which is js. adding those to the shadow edn file won't affect any deps that lein can see.

Reply:
I meant to the compiled JS